src/components/AlertLine.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <li 
+    class="alert" 
+    :style="{ 'background-color': alert.backgroundColor }">
+    <div class="icon-container">
+      <img class="icon" :src="`img/${topic}.svg`" />
+    </div>
+    <div class="alert-message">{{ alert.message }}</div>
```

Comment:
This doesn't respect the `message` formatting we agreed on (separating title, subtitle and sub-subtitle by _@@@_)

Reply:
I talked to @hefowe about that and I think this should be parsed and devided inside the backend when forwarding events.